.devcontainer/set_dot_env.sh

```diff
@@ -8,7 +8,7 @@ readonly file=".env"
 
 echo "Writing ${wanted_line} to ${file}" >&2
 if [[ -f "${file}" ]] && grep --quiet "^${wanted_line_key}=" "${file}"; then
-    sed --in-place "s,^${wanted_line_key}=.*,${wanted_line}," "${file}"
```

Comment:
@brenank, why did you change this?

Reply:
Ah the original sed command only worked on the GNU version, not BSD.

Yeah maybe there's a better solution to make it cross compatible if we can control the env the command is run in, but I'm less familiar with dev containers.

Reply:
I ended up changing to `sed -i`, which is more portable than `--in-place`.

Reply:
Yeah, that's still not cross-platform compatible with BSD sed. I found this article that might have a "better" solution:
https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux
A bit ugly, but I suppose it could work.

Reply:
Yeah, that should work. Changing `-i` with `| tee` should probably work too.